src/components/Navigation/UnauthenticatedNav.js

```diff
@@ -2,9 +2,7 @@ import React from "react";
 import Typography from "@material-ui/core/Typography";
 
 const AuthenticatedNav = () => (
```

Comment:
This is component is dumb. just delete this component and replace `<UnauthenticatedNav/>` in the Navigation component.